tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -805,6 +805,16 @@ internal static class FeatureFlags
             /// </summary>
             public const string OpenTelemetryEnabled = "DD_TRACE_OTEL_ENABLED";
 
+            /// <summary>
+            /// Enables beta support for exporting OTLP metrics generated by the OpenTelemetry Metrics API.
```

Comment:
Two small suggestions here:
- Since we said "experimental" support below, maybe we can change `beta` to `experimental`?
- Can you document that this is only supported for .NET 6 and later? Internally, the reason is that the BCL class `MeterListener` that makes this all possible is shipped in-box starting with [.NET 6](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.metrics.meterlistener)